react/build.sh

```diff
@@ -10,5 +10,5 @@ rm -rf build
 # npm ci does not update minor versions ->
 # (1) less chance of breaking (2) less noise in PR from package-lock.json
 npm ci
-npm run build # defined in 'scripts' in package.json
+CI=false npm run build # defined in 'scripts' in package.json
```

Comment:
nice!
Please remove `unset CI` from deploy.sh which is now redundant + add comment why CI=false
your way is actually better too because it doesn't permanently unset it.